ROMFS/px4fmu_common/init.d-posix/px4-rc.simulator

```diff
@@ -37,6 +37,9 @@ if [ "$PX4_SIMULATOR" = "sihsim" ] || [ "$(param show -q SYS_AUTOSTART)" -eq "0"
 	fi
 
 elif [ "$PX4_SIMULATOR" = "gz" ] || [ "$(param show -q SIM_GZ_EN)" = "1" ]; then
+	# Use Gazebo Gareden or Harmonic
+
+	echo "INFO  [init] Gazebo Garden/Harmonic simulator"
```

Comment:
Let's try to keep it agnostic and not bake in specifics unless we absolutely can't avoid it?

```suggestion
	echo "INFO  [init] Gazebo simulator"
```

Reply:
Agreed!